test/PizzeriaTest.cpp

```diff
@@ -65,3 +64,31 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, testMainFunctionalityForPizzaVariety) {
+    //Given
+    PizzaStub stub{"stub"};
+    StrictMock<PizzaMock> strictPizza{};
+    NiceMock<PizzaMock> nicePizza{};
+    Pizzas pizzas = {&stub, &strictPizza, &nicePizza};
+    EXPECT_CALL(strictPizza, getName()).WillOnce(Return("strictPizza"));
+    EXPECT_CALL(strictPizza, getPrice()).WillOnce(Return(20.0));
+    EXPECT_CALL(strictPizza, getBakingTime()).WillOnce(Return(minutes(2)));
+
+    EXPECT_CALL(nicePizza, getName()).WillOnce(Return("nicePizza"));
+    EXPECT_CALL(nicePizza, getPrice()).WillOnce(Return(40.0));
+    EXPECT_CALL(nicePizza, getBakingTime()).WillOnce(Return(minutes(3)));
+
+    EXPECT_CALL(timerMock, sleep_for(minutes(1))).Times(1);
+    EXPECT_CALL(timerMock, sleep_for(minutes(2))).Times(1);
+    EXPECT_CALL(timerMock, sleep_for(minutes(3))).Times(1);
+
+    //When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    auto price = pizzeria.calculatePrice(orderId);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
+
+    //Then
+    ASSERT_EQ(60, price);
```

Comment:
it's good to have constexpr values like
```cpp
constexpr double strictPrice = 20.0;
constexpr double nicePrice = 40.0;
```
and here you could use `ASSERT_EQ(strictPrice + nicePrice, price)`;

The same for returned minutes and expectations on timerMock :)